shared/naturalcrit/codeEditor/codeEditor.jsx

```diff
@@ -115,6 +125,8 @@ const CodeEditor = createClass({
 				'Shift-Cmd-Enter'  : this.newColumn,
 				'Ctrl-Enter'       : this.newPage,
 				'Cmd-Enter'        : this.newPage,
+				'Ctrl-Alt-F'       : this.findPersistent,
```

Comment:
Is it possible to make persistent Find dialog the default with just `ctrl + f` ?

Reply:
Yep! I'll probs need to un-bind a CodeMirror listener but definitely should be doable!

Reply:
`findPersistent` is now on ctrl+f, and I also bound `findPersistentPrevious` to shift+enter because using ctrl+shift+g was awful.